doc/api/python.rst

```diff
@@ -148,49 +145,40 @@ Each item in a plot's contents is represented by a dictionary. The only mandator
 
-    ``observable``
-        See `Plotting observables`_.
-    ``constraints``
-        See `Plotting constraints`_.
-
-``name`` : string : (optional)
-    The name of the item, for convenience when reporting warnings and errors.
+    - ``observable`` -- See `Plotting observables`_ for the description of this type of plot item.
+    - ``constraints`` -- See `Plotting constraints`_. for the description of this type of plot item.
 
+ * ``name`` (*str*, optional) -- The name of the plot item, for convenience when reporting warnings and errors.
 
 All item types recognize the following optional keys:
 
-``alpha`` : float between 0.0 and 1.0
-    The opacity of the plot item expressed as an alpha value. 0.0 means completely transparent, 1.0 means completely
-    opaque.
-
-``color`` : string containing any valid Matplotlib color specification
-    The color of the plot item. Defaults to one of the colors in the Matplotlib default color cycler.
+ * ``alpha`` (*float*, between 0.0 and 1.0) -- The opacity of the plot item expressed as an alpha value. 0.0 means completely transparent,
+   1.0 means completely opaque.
 
-``label`` : string : (may contain LaTeX commands)
-    The label that appears in the plot's legend for this content item.
+ * ``color`` (*str*, containing any valid Matplotlib color specification) -- The color of the plot item.
+   Defaults to one of the colors in the Matplotlib default color cycler.
+ * ``label`` (*str*, may contain LaTeX commands) -- The label that appears in the plot's legend for this plot item.
 
 Plotting observables
 --------------------
 
-Contents items of type ``observable`` are used to display one of the built-in :class:`observables <eos.Observable>`.
+Contents items of type ``observable`` are used to display one of the built-in `observables <../observables.html>`_.
 The following keys are mandatory:
 
-``observable`` : :class:`qualified name <eos.QualifiedName>`
-    The name of the observable that will be plotted. Must be identify one of the observables known to EOS.
-
-``range`` : list or tuple of two floating point numbers
-    The tuple of (minimal, maximal) values of the specified kinematic variable for which the observable will be evaluated.
+ * ``observable`` (:class:`QualifiedName <eos.QualifiedName>`) -- The name of the observable that will be plotted.
+   Must be identify one of the observables known to EOS; see `the complete list of observables <../observables.html>`_.
+ * ``range`` (*list* or *tuple* of two *float*) --The tuple of [minimal, maximal] values of the specified kinematic variable
+   for which the observable will be evaluated.
 
-Exactly one of the follow keys is mandatory, to specify either a kinematic variable or a parameter to which the x coordinate will be mapped:
+Exactly one of the follow keys is mandatory, to specify either a kinematic variable or a parameter to which the x coordinate
```

Comment:
"of the follow keys" -> "of the following keys" ?!

Reply:
Jupp, thanks!